src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb

```diff
@@ -6,6 +6,19 @@ module Api::Controllers
     class EventsController < BaseController
       EVENT_LIMIT = 200
 
+      get '/:id' do
+        content_type(:json)
+
+        event_id = params[:id]
+        event = Models::Event[event_id]
```

Comment:
not sure if we need to turn it back into integer: https://github.com/cloudfoundry/bosh/blob/master/src/bosh-director/lib/bosh/director/api/event_manager.rb#L12 ?